back/src/Services/SocketManager.ts

```diff
@@ -430,6 +417,9 @@ export class SocketManager {
                     if (!meetingId) {
                         throw Error("Meeting is not created");
                     }
+                    if (!meetingLink) {
```

Comment:
I  would leave only one check on the place and delete ```                    if (!meetingId) {
                        throw Error("Meeting is not created");
                    }```

Reply:
If that error is removed, the linter complains that meetingLink in line 435 might be undefined. Would this be an alright compromise? 

```
if (!meetingId || !meetingLink) {
                        throw Error("Meeting is not created");
                    }```